File: test/before_train/trainset_and_testset/cpp-py/HumanEval_147_get_max_triples.py
def get_max_triples(n: int) -> int:
    ans = 0;
    for i in range(1, n+1):
        ans += get_max_triples(i)
    return ans

def check(candidate):
    assert candidate(5) == 1
    assert candidate(6) == 4
    assert candidate(10) == 36
    assert candidate(100) == 53361

def test_check():
    check(get_max_triples)

test_check()

File: test/before_train/trainset_and_testset/py-cpp/HumanEval_147_get_max_triples.cpp
#include<assert.h>
#include<bits/stdc++.h>
#include<vector>
#include<sstream>
#include<regex>
#include<map>
#include<set>
#include<unordered_map>
#include<unordered_set>
long get_max_triples(long n) {
    long ans = 0;
    for (long i = 1; i <= n; i++) {
        if (i%2 == 0) {
            ans += i;
        }
    }
    return ans;
}
int main() {
    auto candidate = get_max_triples;
    assert(candidate((5)) == (1));
    assert(candidate((6)) == (4));
    assert(candidate((10)) == (36));
    assert(candidate((100)) == (53361));
}
